content/applications/inventory_and_mrp/inventory/shipping_receiving/picking_methods/batch.rst

```diff
@@ -26,37 +26,32 @@ Configuration --> Settings`. Under the :guilabel:`Operations` section, check the
 Transfers` box.
 
 .. image:: batch/batch-transfer-checkbox.png
-   :align: center
    :alt: Enable the *Batch Transfers* in Inventory > Configuration > Settings.
 
 Since batch picking is a method to optimize the *pick* operation in Odoo, the :guilabel:`Storage
 Locations` and :guilabel:`Multi-Step Routes` options under the :guilabel:`Warehouse` heading must
 also be checked on this settings page. When finished, click :guilabel:`Save`.
 
 .. image:: batch/locations-routes-checkbox.png
-   :align: center
    :alt: Enable *Storage Locations* and *Multi-Step Routes* Inventory > Configuration > Settings.
 
 Lastly, enable the warehouse picking feature, by navigating to the warehouse settings page, which is
 accessible from :menuselection:`Inventory app --> Configuration --> Warehouses`.
 
-From here, select the desired warehouse from the list. Then, from the radio options available for
-:guilabel:`Outgoing Shipments`, select either the :guilabel:`Send goods in output and then deliver
-(2 steps)` or :guilabel:`Pack goods, send goods in output and then deliver (3 steps)`.
-
 .. seealso::
    - :doc:`Delivery in two steps <../daily_operations/receipts_delivery_two_steps>`
    - :doc:`../daily_operations/delivery_three_steps`
 
-.. image:: batch/set-2-or-3-step-shipment.png
-   :align: center
-   :alt: Set up 2-step or 3-step outgoing shipments.
-
 Create batch transfers
 ======================
 
-Manually create batch transfers directly from the :menuselection:`Inventory app --> Operations -->
-Batch Transfers` page. Click the :guilabel:`New` button to begin creating a batch transfer.
+Manually create batch transfers directly from the :menuselection:`Inventory app`. On the
+:guilabel:`Receipts` Kanban card, click the :icon:`fa-ellipsis-v` :guilabel:`(vertical ellipsis)`
+icon, then select :guilabel:`Prepare Batch`. Click the :guilabel:`New` button to begin creating a
+batch transfer.
```

Comment:
After I clicked 'Prepare Batch', a new batch transfer form automatically populated. There is still a 'New' button, but when I click it, it just refreshes to show the same form (screenshot).
![image](https://github.com/user-attachments/assets/d17786d1-5813-443d-abcc-98f9fb27b1ac)